src/Tools/CompilerBenchmarks/FixedCsProjGenerator.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.CsProj;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+
+namespace CompilerBenchmarks
+{
+    public class FixedCsProjGenerator : CsProjGenerator
+    {
+        private const string Template = @"
+<Project ToolsVersion=""15.0"">
+  <PropertyGroup>
+    <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
+    <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
```

Comment:
Consider also adding the other suppressions inside 

[https://github.com/dotnet/roslyn/blob/master/src/Workspaces/CoreTestUtilities/Resources/Directory.Build.props](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/CoreTestUtilities/Resources/Directory.Build.props)